lightbluetent/templates/admin/admin.html

```diff
@@ -51,18 +52,30 @@
                 {% for n in range(2) %}
                 {{social_field(n, social_defaults[n])}}
                 {% endfor %}
-                <small class="form-text text-muted">Add multiple URLs and emails to spruce up your main page.</small>
+                <small class="form-text text-muted">{{ _("Add multiple URLs and emails to spruce up your main page.") }}</small>
+            </div>
+            {% endif %}
+            {% if has_directory_page %}
+            <div class="form-group-row">
```

Comment:
Have a look at the other form groups. You'll notice that they have been changed in the latest PR. `form-group-row` is a non-existent class. You just need `form-group`. Take some hints from the text area for the full description.